api/src/main/java/org/openmrs/module/openconceptlab/client/OclClient.java

```diff
@@ -116,6 +112,10 @@ public OclResponse fetchOclConcepts(String url, String token) throws IOException
     public GetMethod executeExportRequest(String url, String collectionVersion) throws IOException{
 
 		GetMethod exportUrlGet = getExportUrl(url, collectionVersion);
+		if (!StringUtils.isBlank(token)) {
+			get.addRequestHeader("Authorization", "Token " + token);
+			get.addRequestHeader("Compress", "true");
+		}

```

Comment:
```suggestion
		if (!StringUtils.isBlank(token)) {
			exportUrlGet.addRequestHeader("Authorization", "Token " + token);
		}

```

Note that we're also going to need to pass the token into this function.